### DISCUSSION TITLE
GitHub Pages deploy but website is blank

### DISCUSSION THREAD

**Discussion by LapoBini** [community/community#139233] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi, 
I am trying to build my webpage on GitHub Pages. I know nothing about HTML, but I built my academic website using Blogdown and Rmarkdown. 

If I run my webpage locally, it works fine with all the sections that I want, files, etc.; however, when I deploy on Github Pages it is completely blank, and the page title becomes index.knit.
If I deploy the website in netlify.com, it works fine. 

The repository is here https://github.com/LapoBini/lapobini.github.io, and your help is highly appreciated.

**Comment by LiteBrite82** [comment, marked as answer]
Hi @LapoBini 
I'm not familiar with Blogdown, but I found an article that may be helpful: https://www.r-bloggers.com/2022/01/hosting-a-blogdown-blog-on-github-pages/

**Comment by abdessalems** [comment]
The user is building a webpage on GitHub Pages using Blogdown and Rmarkdown, but the site appears blank after deployment, with the page title showing "index.knit." It works fine locally and on Netlify. They are asking for help with this issue and have shared their GitHub repository for troubleshooting
